fdbclient/include/fdbclient/Tenant.h

```diff
@@ -59,10 +59,14 @@ struct TenantMapEntryTxnStateStore {
 	int64_t id = -1;
 	TenantName tenantName;
 	TenantAPI::TenantLockState tenantLockState = TenantAPI::TenantLockState::UNLOCKED;
+	Optional<UID> tenantLockId;
```

Comment:
why do we need the id in the txnStateStore? I think we only need this to prevent race conditions when changing the lock state from the client, but I don't think the proxy is needing this.

Reply:
You're right, I added this locally at some point last week when I was getting started on this and forgot to verify whether we actually need it here